public/js/graph/react_node.js.jsx

```diff
@@ -50,26 +50,61 @@ function getNodes(mode){
         entry['style'] = getStyles(entry['attributes']['style']);
         entry['children'] = [];
         //value.children is an HTML collection, converting to array here
-        var children_arr = Array.prototype.slice.call(element.children);
+        var childrenArr = Array.prototype.slice.call(element.children);
         //Assumed only one level of children, the <rect> and one or more <text>
-        children_arr.forEach(function(child) {
-            var child_entry = {};
-            child_entry['attributes'] = getAttributes(child.attributes);
-            child_entry['style'] = getStyles(child_entry['attributes']['style']);
+        childrenArr.forEach(function(child) {
+            var childEntry = {};
+            childEntry['attributes'] = getAttributes(child.attributes);
+            childEntry['style'] = getStyles(childEntry['attributes']['style']);
             //innerHTML is just for text within the <text>
             //there aren't anymore children since it was assumed only one level of children
-            child_entry['innerHTML'] = child.innerHTML;
-            entry['children'].push(child_entry);
+            childEntry['innerHTML'] = child.innerHTML;
+            entry['children'].push(childEntry);
         });
-        dict_list.push(entry);
+        dictList.push(entry);
     });
-    return dict_list;
+    return dictList;
 }
 
+//Quick fix to initiallyTakeable case-sensitivity
```

Comment:
@ryanfan Just remove the "initially takeable" code for now. I don't care if nothing is initially takeable.